src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -451,13 +456,18 @@ private IEnumerable<MemberDeclarationSyntax> NamespaceMembers
     {
         get
         {
-            IEnumerable<MemberDeclarationSyntax> result =
-                from entry in this.committedCode.MembersByModule
-                select ClassDeclaration(Identifier(this.options.ClassName))
+            var first = this.committedCode.MembersByModule.Select(entry => ClassDeclaration(Identifier(this.options.ClassName))
```

Comment:
I'd like to see this change reworked so we are not repeating so much code as this was already complex. Also, as this turns out to be an expensive part CPU-wise to the generation, I don't want to use Skip(1) which will "realize" the first (and often only) element a second time. If we need to rewrite this as a loop instead of using linq extension methods, so be it.